apps/angular/src/app/features/anime/anime-details/anime-details.component.ts

```diff
@@ -49,6 +71,31 @@ export class AnimeDetailsComponent {
 		);
 	}
 
+	/**
+	 * Open modal for anime deletion confirmaion.
+	 * @param id - Anime id.
+	 */
+	protected openDeletionConfirmation(id: number): void {
```

Comment:
To give additional hints about identifiers (or other properties) you can use TypeScript.
```suggestion
	protected openDeletionConfirmation(id: AnimeDetails['id']): void {
```